app/templates/gulp/_scripts.js

```diff
@@ -16,31 +16,21 @@ gulp.task('scripts',
 <% } %>
 
   return gulp.src(paths.src + '/{app,components}/**/*.<%= props.jsPreprocessor.extension %>')
-<% if (props.jsPreprocessor.extension === 'js') { %>
-    .pipe($.jshint())
-    .pipe($.jshint.reporter('jshint-stylish'))
-<% } if (props.jsPreprocessor.key !== 'none') { %>
```

Comment:
I remove this line because it's dead code, as `gulp/script.js` won't be included if `props.jsPreprocessor.key === 'none'`.